testsuite/MDAnalysisTests/coordinates/test_h5md.py

```diff
@@ -2,7 +2,12 @@
 from numpy.testing import assert_almost_equal, assert_array_equal
 import numpy as np
 import MDAnalysis as mda
-from MDAnalysis.coordinates.H5MD import HAS_H5PY
```

Comment:
I liked getting HAS_H5PY from the module better, so you only have the same code once.

Or is there a reason that you _must_ check the import here separately?

Reply:
No reason, I just thought testing directly again would be better for some reason. I'll change it back really quick

Reply:
Ah, wait there was a reason why I added that. Previously, I wasn't importing h5py at all, but the additional tests I wrote needed `h5py.File` so I threw the import check in. But you're right, it's not necessary